.github/CONTRIBUTING.md

```diff
@@ -1,4 +1,4 @@
 Contributing
 ======
 
-Information on contributing to this repo is in the [Contributing Guide](https://github.com/aspnet/Home/blob/master/CONTRIBUTING.md) in the Home repo.
+Information on contributing to this repo is in the [Contributing Guide](https://github.com/aspnet/Home/blob/master/CONTRIBUTING.md) in the AspNetCore repo.
```

Comment:
Scaffolding

Reply:
The link is pointing to the AspNetCore repo so we should name it such. I can switch it to dotnet/AspNetCore for more clarity.